easybuild/easyconfigs/r/R/liquidSVM-1.2.4-fix_ppc_build.patch

```diff
@@ -41,7 +41,7 @@ diff -aur liquidSVM-orig/src/sources/shared/system_support/memory_allocation.h l
  // Check for GCC
  #if __GNUC__
 -	#if __x86_64__ || __ppc64__
-+	#if __x86_64__ || __ppc64__ || __PPC64__
++	#if __x86_64__ || __ppc64__ || __PPC64__ || __aarch64__
```

Comment:
Please update the patch description (and filename) to clarify it also covers `aarch64`?

Reply:
Ack. Updated patch description and filename.